rskj-core/src/test/java/org/ethereum/datasource/KeyValueDataSourceTest.java

```diff
@@ -34,13 +34,13 @@ public static Collection<Object[]> data() throws IOException {
         Path tmpDir = Files.createTempDirectory("rskj");
         return Arrays.asList(new Object[][] {
             { new HashMapDB(), HashMapDB.class.getSimpleName(), true},
-            { new LevelDbDataSource("test", Files.createTempDirectory(tmpDir, "default").toString()), LevelDbDataSource.class.getSimpleName(), true },
+            { new RocksDbDataSource("test", Files.createTempDirectory(tmpDir, "default").toString()), RocksDbDataSource.class.getSimpleName(), true },
```

Comment:
I'd rather return here data sources for both leveldb and rocksdb. We're still supporting `LevelDbDataSource`

Reply:
Done